lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -30,7 +30,7 @@
 <!-- Note: these will only fail when using the static-server.js, which supports the ?delay=true param.
      If you're using your own server, the resource will load instantly and the
      stylesheets will be ignored for being below the threshold. -->
-<link rel="stylesheet" href="./dobetterweb/dbw_tester.css?delay=100"> <!-- FAIL, when run under smokehouse -->
+<link rel="stylesheet" href="./dbw_tester.css?delay=100"> <!-- FAIL, when run under smokehouse -->
```

Comment:
> also, I wish we put the audit id that these failed for.

is it possible to check this and add a note since it's being changed anyways? Presumably by the comment something currently failing should pass if there isn't a delay

Reply:
it actually didn't really fail anything (only failures were b/c the name changes) ...

Reply:
> it actually didn't really fail anything (only failures were b/c the name changes) ...

make sense to remove the `<!-- FAIL, when run under smokehouse -->` then?

Reply:
ya done